### PR TITLE
Add DaSilva.org.br acessibility checker

### PR DESCRIPTION
**Pull request by cassiosantos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ List of links to the various checkers out there on the web for sites, domains, s
 | https://tenon.io/ | Test sites for accessibility issues and offer solutions for problems found. | Free and paid plans |
 | https://www.toptal.com/designers/colorfilter | Checks pages for color accessibility. | Free |
 | http://wave.webaim.org/ | Gives a detailed report about markup and accessibility concerns including ARIA roles. Provides solutions for problems found. | Free |
+| http://www.dasilva.org.br/ | Test sites for accessibility issues (W3C/WAI  - WCAG1 and WCAG2 - and E-mag). | Free |
```
Comment (Brunty) [inline README.md]: Can this be put in alphabetical order as per the [Contributing Guidelines](https://github.com/Brunty/awesome-checker-services/blob/master/CONTRIBUTING.md) (It's alphabetical on the name of the service, so `dasilva` is a `d`)

**Comment by Brunty** [comment]
This raises the question of potentially needing to show locales and languages for services as well - not in the scope of this PR, but something that I'll raise an issue for now.

File: README.md (hunk 1 above)
Comment (cassiosantos) [reply README.md]: Done

**Comment by Brunty** [comment]
Thank you! :)

**Pull request merged**